packages/components/src/Tooltip/Tooltip.tsx

```diff
@@ -1,45 +1,39 @@
-import { ReactNode } from 'react';
-
-import { useTooltip } from '@react-aria/tooltip';
+import type { ReactNode } from 'react';
+import { OverlayArrow, Tooltip } from 'react-aria-components';
+import type RAC from 'react-aria-components';
 
 import { cn, useClassNames } from '@marigold/system';
-import { HtmlProps } from '@marigold/types';
 
-import { useTooltipContext } from './Context';
 import { TooltipTrigger } from './TooltipTrigger';
 
-// Props
-// ---------------
-export interface TooltipProps extends Omit<HtmlProps<'div'>, 'className'> {
+type RemovedProps = 'className' | 'isOpen' | 'style';
+
+export interface TooltipProps extends Omit<RAC.TooltipProps, RemovedProps> {
   children?: ReactNode;
   variant?: string;
   size?: string;
+  open?: RAC.TooltipProps['isOpen'];
 }
 
-// Component
-// ---------------
-export const Tooltip = ({ children, variant, size }: TooltipProps) => {
-  const { arrowProps, state, overlayRef, placement, ...rest } =
-    useTooltipContext();
```

Comment:
Because we don't use useTooltipContext anymore the file Tooltip/Context.ts can be removed. Then codecov should also be happy.